lib/qa/authorities/local/file_based_authority.rb

```diff
@@ -8,13 +8,17 @@ def initialize(subauthority)
     def search(q)
       r = q.blank? ? [] : terms.select { |term| /\b#{q.downcase}/.match(term[:term].downcase) }
       r.map do |res|
-        { id: res[:id], label: res[:term] }.with_indifferent_access
+        qa_res = { id: res[:id], label: res[:term] }.with_indifferent_access
+        qa_res[:uri] = res[:uri] if res.key?(:uri)
+        qa_res
```

Comment:
This works, but there is a more shorthand way to write this:
```ruby
{ id: res[:id], label: res[:term], uri: res[:uri] }.compact.with_indifferent_access
```

Reply:
Good to know.  I'll update.